lib/bolt/transport/local_windows.rb

```diff
@@ -166,7 +166,7 @@ def run_task(target, task, arguments, options = {})
           if Powershell.powershell_file?(script) && stdin.nil?
             command = Powershell.run_ps_task(arguments, script, input_method)
             command = environment_params + Powershell.shell_init + command
-            interpreter ||= 'powershell.exe'
+            interpreter ||= ['powershell.exe', *Bolt::Transport::Powershell.singleton_class::PS_ARGS]
```

Comment:
I think we can just make `PS_ARGS` a method in the `Bolt::Transport::Powershell` module. It is meant to make powershell specific code/templates available where needed. I think having to access `PS_ARGS` with `singleton_class` is not ideal. I think we can take this over and assign a ticket to it. Thanks @McdonaldSeanp